src/config/ConfigManager.cpp

```diff
@@ -279,8 +306,8 @@ CConfigManager::CConfigManager() {
     m_pConfig->addConfigValue("general:border_size", {1L});
     m_pConfig->addConfigValue("general:no_border_on_floating", {0L});
     m_pConfig->addConfigValue("general:border_part_of_window", {1L});
-    m_pConfig->addConfigValue("general:gaps_in", {5L});
-    m_pConfig->addConfigValue("general:gaps_out", {20L});
+    m_pConfig->addConfigValue("general:gaps_in", Hyprlang::CConfigCustomValueType{&configHandleGapSet, configHandleGapDestroy, "5"});
```

Comment:
why are you mixing `&func` and `func`? They are the same, pick one. Probably `func`